src/Pelagos/Util/UrlValidation.php

```diff
@@ -20,8 +20,12 @@ class UrlValidation
     public function validateUrl($url)
     {
         $ch = curl_init();
+        $cookies = tempnam('/tmp','cookie.txt');
+        curl_setopt($ch, CURLOPT_COOKIEJAR, $cookies);
+        curl_setopt($ch, CURLOPT_COOKIEFILE, $cookies);
```

Comment:
When I tried this only cookiejar is needed.

Reply:
Done. Changed it